libs/pages/summary/src/lib/ui/summary.tsx

```diff
@@ -1,62 +1,226 @@
 import { useTranslation } from 'react-i18next';
 import { BreadCrumbs } from '@yadoms/shared';
 import { loadSystemInformations } from '../summary-api';
 import { useQuery } from '@tanstack/react-query';
+import React, { ReactNode } from 'react';
+import { Container } from '@mantine/core';
+import semver from 'semver/preload';
+import classes from './summary.module.css';
+import {
+  DatabaseIcon, DatabaseVersionIcon,
+  LinuxLogo,
+  MacOSLogo,
+  PostgreSQLLogo, PowerIcon,
+  SQLIcon,
+  SQLiteLogo,
+  SystemIcon, VersionIcon,
+  WindowsLogo,
+  SizeIcon
+} from './icons';
 
-export function Summary() {
+
+
+const customization = {
+  Window: {color: "#4057dc", size:40},
+  Linux:{color: "#6f706e", size:40},
+  MacOs:{color: "#000000", size:40},
+  DefaultOs:{color: "#6e6e70", size:40},
+  SqLite:{color: "#66b2fc", size:40},
+  Postgres:{color: "#3177ff", size:40},
+  Database:{color: "#f14809", size:40},
+  DatabaseVersion: {color: "#cec483", size:40},
+  Power: {color: "#00ff28" ,size:40},
+  Version: {color: "#873be1", size:40},
+  Size: {color: "#e52121", size:40},
+  SQL: {color: "#0048fd", size:40},
+
+
+
+}
+export function isVersion(version: string) : boolean {
+  return semver.valid(version) != null;
+}
+
+
+
+function formatDate(isoDate: string | undefined): string {
+  const {i18n } = useTranslation();
+  const locale = i18n.language;
+  if (!isoDate) {
+    return '';
+  }
+
+  const year = parseInt(isoDate.slice(0, 4), 10);
+  const month = parseInt(isoDate.slice(4, 6), 10) - 1;
+  const day = parseInt(isoDate.slice(6, 8), 10);
+  const hour = parseInt(isoDate.slice(9, 11), 10);
+  const minute = parseInt(isoDate.slice(11, 13), 10);
+  const second = parseInt(isoDate.slice(13, 15), 10);
+  const millisecond = Math.round(parseFloat("0." + isoDate.split('.')[1]) * 1000);
+
+  const date = new Date(year, month, day, hour, minute, second, millisecond);
+
+  const options: Intl.DateTimeFormatOptions = {
+    weekday: 'long', year: 'numeric', month: 'long', day: 'numeric',
+    hour: 'numeric', minute: 'numeric', second: 'numeric',
+  };
+
+  return date.toLocaleDateString(locale, options);
+
+}
+
+export function Card({ icon, label, content, color, isLoading, children }: {
+  icon: ReactNode,
+  label: string,
+  content: string | number | undefined,
+  color: string,
+  isLoading: boolean,
+  children: ReactNode
+}) {
+  return (
+    <Container mih={100}  p={'xl'} withBorder className={classes.item} styles={(theme) => ({
+      root: {
+        boxShadow: `0px 0px 20px ${color}`,
+        border: `1px solid ${color}`
+      },
+    })}>
+      <Center>
+        {icon}
+      </Center>
+      <Text
+        weight={500}
+        size={{ base: 'xs', sm: 'lg' }}
+      >
+        {label}
+      </Text>
+      {isLoading ? (
+        <Skeleton height={20} mt="md" width="80%" radius="sm" />
+      ) : isVersion(content) ? (
+        <Badge color="pink" variant="light" mt={10}>
```

Comment:
All data comes from one Ip and calls onces could create a specific skeleton for all card and move this logic of displaying the loader into the parent caller